profiles/base/make.defaults

```diff
@@ -127,10 +127,10 @@ BOOTSTRAP_USE="unicode internal-glib pkg-config split-usr python_targets_python3
 
 # Mike Gilbert <floppym@gentoo.org> (2012-05-15)
 # Default target(s) for python-r1.eclass
-# Mikle Kolyada <zlogene@gentoo.org> (2020-05-06)
-# Updated to python3_7
-PYTHON_TARGETS="python2_7 python3_7"
-PYTHON_SINGLE_TARGET="python3_7"
+# Mikle Kolyada <zlogene@gentoo.org> (2020-12-02)
+# Updated to python3_8
+PYTHON_TARGETS="python2_7 python3_8"
+PYTHON_SINGLE_TARGET="python3_8"
```

Comment:
Synced BOOTSTRAP_USE in 2cbbf8a6271301b928d5c1d5dce435c2a291c8ae.

Reply:
@zmedico, ugh, thanks, forgotten one